vespa-athenz/src/main/java/com/yahoo/vespa/athenz/identityprovider/client/AthenzCredentialsService.java

```diff
@@ -59,15 +60,15 @@ AthenzCredentials registerInstance() {
                 document.ipAddresses(),
                 keyPair);
 
-        try (com.yahoo.vespa.athenz.client.zts.ZtsClient ztsClient =
+        try (ZtsClient ztsClient =
                      new DefaultZtsClient(URI.create(identityConfig.ztsUrl()), nodeIdentityProvider)) {
             InstanceIdentity instanceIdentity =
                     ztsClient.registerInstance(
                             new AthenzService(identityConfig.configserverIdentityName()),
                             tenantIdentity,
                             null,
                             EntityBindingsMapper.toAttestationData(document),
-                            true,
+                            false,
```

Comment:
remember to create a new registerinstance that does not require _null_ and _false_